package.json

```diff
@@ -40,11 +40,15 @@
     "ip": "1.1.5",
     "konfig": "0.2.1",
     "optional": "0.1.4",
+    "swagger-tools": "^0.10.3",
```

Comment:
Use the exact version to stop minor changes effecting builds

Reply:
Good spot. Done.